workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -54,12 +54,12 @@ public abstract class WorkerConnector : MonoBehaviour, IDisposable
         private static readonly SemaphoreSlim WorkerConnectionSemaphore = new SemaphoreSlim(1, 1);
 
         // Important run in this step as otherwise it can interfere with the the domain unloading logic.
-        private void OnApplicationQuit()
+        protected void OnApplicationQuit()
         {
             Dispose();
         }
 
-        private void OnDestroy()
+        protected void OnDestroy()
```

Comment:
made those two methods protected as you might want to override them in user code